tests/fixtures/datasets.py

```diff
@@ -28,6 +28,12 @@ def trackmate_data_dir(pytestconfig):
     return os.path.join(pytestconfig.rootdir, "tests/data/microscopy/trackmate")
 
 
+@pytest.fixture
+def cell_tracking_data_dir(pytestconfig):
+    """Dir path to cell tracking challenge data."""
+    return os.path.join(pytestconfig.rootdir, "tests/data/microscopy/cell_tracking")
```

Comment:
```suggestion
    return Path(pytestconfig.rootdir) / "tests/data/microscopy/cell_tracking"
```
prefer `pathlib` over `os.path`